Nim/Program.cs

```diff
@@ -0,0 +1,42 @@
+﻿Console.WriteLine("Welcome to Nim!\nLet's play: we take turns drawing matches, and the one who draws the last one is the loser.");
+int matches = 24;
+string draw = "||||||||||||||||||||||||";
+Console.WriteLine($"{draw} ({matches})");
+Console.WriteLine("How many matches do you want to draw?");
+
+Play:
+// User turn
+int userInput = Convert.ToInt32(Console.ReadLine());
+if (userInput == 1 || userInput == 2 || userInput == 3){
```

Comment:
Consider using only two conditions: if userInput greater than 0 AND user input less than 4. Less work for the computer :)